buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -319,6 +334,8 @@ class ClusterFormationTasks {
     static Task configureCopyPluginsTask(String name, Project project, Task setup, NodeInfo node) {
         if (node.config.plugins.isEmpty()) {
             return setup
+        } else if (node.nodeVersion != VersionProperties.elasticsearch) {
+            return configureCopyBwcPluginsTask(name, project, setup, node)
```

Comment:
I think we should have a method `configureCopyPluginsTask` and another method `configureCopyBwcPluginsTask` - the condition to call one or the other should be done at a higher level